CI worklow fails due AWS credential expiration

Sometimes, workflow jobs fails if its duration takes long time  
Example error message from [job](https://github.com/elastio/elastio-snap/actions/runs/5573677218/jobs/10181284409):
```
Run excl_ptrn="*GPG-KEY"
+ aws s3api put-object --acl private --body repobuild/artifacts/Amazon/2023/aarch64/Packages/elastio-snap-debuginfo-0.12.2-1.amzn2023.aarch64.rpm --bucket artifacts.assur.io --key linux/elastio-snap/master/1126/rpm/Amazon/2023/aarch64/Packages/elastio-snap-debuginfo-0.12.2-1.amzn2023.aarch64.rpm

An error occurred (ExpiredToken) when calling the PutObject operation: The provided token has expired.
+ exit 7
Error: Process completed with exit code 7.
```

Thanks @skypodolsky  for reporting

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -82,6 +82,7 @@ jobs:
         with:
           aws-region: ${{ env.AWS_DEFAULT_REGION }}
           role-to-assume: ${{ secrets.ASSURIODEV_OIDC_AWS_ROLE_ARN }}
+          role-duration-seconds: 10800  # 3 hours
 
       - name: Start a box
         if: always()

